Service/SpClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Italia\SpidSymfonyBundle\Service;
+
+use Italia\Spid\Sp;
+
+class SpClient extends Sp
```

Comment:
I still didn't find the time to test this, but if it works like this then both the explicit constructor and the `getConfiguredSPClient` method are redundant.
Let's remove this class altogether and point the service to the `Italia\Spid\Sp` class directly

Reply:
ok